src/_guides/libraries/objective-c-interop.md

```diff
@@ -293,6 +293,58 @@ then check the status, and wait for the duration of the audio file:
     }
 ```
 
+### Callbacks and multithreading limitations
+
+Multithreading issues are the biggest limitation
+of Dart's experimental support for Objective-C interop.
+While `ffigen` supports converting
+Dart functions to Objective-C blocks,
+these limitations must be kept in mind.
```

Comment:
Having another sentence between "these limitations" and the list of limitations makes this hard to follow, I think. How about folding this whole sentence into the list, since it only actually applies to part of it? The second bullet point could become:

> While `ffigen` supports converting Dart functions to Objective-C blocks, most Apple APIs don't have any guarantees about the the thread on which a callback will run.

instead of having this sentence in the intro at all.

Reply:
Done